src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -8215,31 +8268,65 @@ void checkWellKnown(WellKnownMember member)
                             original.Parameters[0] is { Type: { SpecialType: SpecialType.System_Int32 }, RefKind: RefKind.None } &&
                             original.Parameters[1] is { Type: { SpecialType: SpecialType.System_Int32 }, RefKind: RefKind.None })
                         {
-                            patternSymbol = method;
+                            method.AddUseSiteInfo(ref useSiteInfo);
+                            diagnostics.ReportUseSite(method, syntax);
+                            ReportDiagnosticsIfObsolete(diagnostics, method, syntax, hasBaseReceiver: false);
```

Comment:
>ReportDiagnosticsIfObsolete(diagnostics, method, syntax, hasBaseReceiver: false); [](http://example.com/codeflow?start=28&length=81)

Why are we still doing this manually here?

Reply:
Removed. `ReportDiagnosticsIfObsolete(diagnostics, method, syntax, hasBaseReceiver: false);` is now handled by `BindMethodGroupInvocation` in `makeCall`.